python-backend/app/api/variances/resources/variance_resource.py

```diff
@@ -23,14 +22,32 @@ class VarianceResource(Resource, UserMixin, ErrorMixin):
             'per_page': f'The number of records to return per page. Default: {PER_PAGE_DEFAULT}',
             'variance_application_status_code':
             'Comma-separated list of code statuses to include in results. Default: All status codes.',
-        })
+            'compliance_codes':'Comma-separated list of compliance codes to be filtered. Default: All compliance codes',
+            'major': 'boolean indicating if variance is from a major or regional mine',
+            'region': 'Comma-separated list of regions the mines associated with the variances are located in',
+            'issue_date_before': 'Latest possible issue date returned (inclusive)',
+            'issue_date_after': 'Earliest possible issue date returned (inclusive)',
+            'expiry_date_before': 'Latest possible expiry date returned (inclusive)',
+            'expiry_date_after': 'Earliest possible expiry date returned (inclusive)'})
     @requires_any_of([VIEW_ALL])
     @api.marshal_with(PAGINATED_VARIANCE_LIST, code=200)
     def get(self):
-        records, pagination_details = self._apply_filters_and_pagination(
-            page_number=request.args.get('page', PAGE_DEFAULT, type=int),
-            page_size=request.args.get('per_page', PER_PAGE_DEFAULT, type=int),
-            application_status=request.args.get('variance_application_status_code', type=str))
+        args={
```

Comment:
I'm not so sure the args dict needs to be there, it seems to me that you can just pass down request.args to apply_filters_pagination method directly and access values there.

Alternatively, if you're unpacking all the args beforehand, you can also pass them as an individual parameter to the function.